java/src/jmri/jmrix/lenz/li100/LI100XNetInitializationManager.java

```diff
@@ -14,11 +14,12 @@
 @Deprecated
```

Comment:
This class is marked as deprecated. Why not remove it entirely? If unsure, we could ask Paul Bender (pabender).

Reply:
> This class is marked as deprecated. Why not remove it entirely? If unsure, we could ask Paul Bender (pabender).
I'm working through these systematically, working on various member functions first.  I'll get to it, but I'm trying to keep things simple in each of these PRs because I don't trust my ability to keep everything in my head.